src/main/java/cubicchunks/FlatCubicChunksWorldType.java

```diff
@@ -40,12 +40,20 @@ public FlatCubicChunksWorldType() {
 
 	@Override public void registerWorldGen(ICubicWorldServer world, GeneratorPipeline pipeline) {
 		ServerCubeCache cubeCache = world.getCubeCache();
+		
 		// init the worldgen pipeline
-		pipeline.addStage(GeneratorStage.TERRAIN, new FlatTerrainProcessor(cubeCache, 5));
-		pipeline.addStage(GeneratorStage.SURFACE, new NullProcessor("Surface", cubeCache));
-		pipeline.addStage(GeneratorStage.FEATURES, new NullProcessor("Features", cubeCache));
-		pipeline.addStage(GeneratorStage.LIGHTING, new FirstLightProcessor("Lighting", cubeCache, 5));
-		pipeline.addStage(GeneratorStage.POPULATION, new NullProcessor("Population", cubeCache));
+		GeneratorStage terrain = new GeneratorStage("TERRAIN");
+		GeneratorStage surface = new GeneratorStage("SURFACE");
+		GeneratorStage features = new GeneratorStage("FEATURES");
+		GeneratorStage lighting = new GeneratorStage("LIGHTING");
+		GeneratorStage population = new GeneratorStage("POPULATION");
+		
+		pipeline.addStage(terrain, new FlatTerrainProcessor(cubeCache, 5));
+		pipeline.addStage(surface, new NullProcessor("Surface", cubeCache));
```

Comment:
Stages with NullProcessor can be safely removed. I added them just to workaround the fact that all stage processors had to exist and be registered.